### ISSUE TITLE
[BUG] Apple ProRes MOV with CEA-608 track has wrong offset

### ISSUE THREAD

**Issue by rayfoss** [CCExtractor/ccextractor#1095]
- [x] I am an active contributor to CCExtractor.

**Necessary information**
- What platform did you use? - [x] Linux

https://storage.googleapis.com/learn-video/Testing/Teeny%20Test.zip
[raw eic track dump.zip](https://github.com/CCExtractor/ccextractor/files/3283236/raw.eic.track.dump.zip)

===============================================

This has stumped me. Decoding Final Cut CEA-608 from Master ProRes 422 MOV exports to srt WORKS fine... except the offset is all wrong... the subtitles start exactly at 00:00:01,569

The only error is this:
```
Processing track 4, type=clcp subtype=c608
100%  |  59:33Processing track 5, type=text subtype=text
Unsupported track type text:text! Please report.
  0%  |  00:00
Unsupported track type text:text! Please report.
  3%  |  00:51
```
Full: https://pastebin.com/Y9gndLw4

**Comment by vikramcs** [comment]
There are many high-end hardware and software solutions for encoding closed captions for broadcast purposes, but if you are using a file-based workflow Apple Compressor is an easy and cost-effective option.
This tutorial describes how to add CEA-608/708 closed captions to video using Compressor and an SCC output file from CaptionSync.
https://youtu.be/-r59sSiDBKU

**Comment by cfsmp3** [comment]
@rayfoss does this happen with all files, or just a specific one? 
Of all of them, is it all of them in general or all of them only if they come from Final Cut?
Do the subs play in sync when played with VLC or ffplay?

**Comment by rayfoss** [comment]
Happened with all of them... They all come from final cut. We needed to use
that format for DVD authoring :/

**Comment by cfsmp3** [comment]
But is this problem specific to CCExtractor or also common to VLC, etc?
Maybe final pro is just breaking the 608 data?

**Comment by rayfoss** [comment]
Quicktime is the only player I know who can read it... aside from compressor/final cut. Which is extra lame. When it does, it is accurate.

**Comment by rayfoss** [comment]
I would agree... if Quicktime and every Apple product that supports it didn't work seamlessly :/

**Comment by cfsmp3** [comment]
@FossPrime Do you still have this sample, or any other with the same problem?

**Comment by FossPrime** [comment]
No and we no longer use that caption format.
